clients/privacy-center/cypress/support/commands.ts

```diff
@@ -181,6 +181,15 @@ declare global {
   interface Window {
     fidesConfig?: FidesConfig;
     dataLayer?: Array<any>;
+    __tcfapi: (
+      command: string,
+      version: number,
+      // tcData should be type TCData from the IAB's TCF library.
+      // We could consider adding that library here, or reexporting just the type
+      // from fides-js
+      callback: (tcData: any, success: boolean) => void,
```

Comment:
Is it any extra work to export the type? Or is that not working for some reason? otherwise, `any` is fine for testing purposes

Reply:
so I think we'd have to export it from `fides-tcf`, and that's not currently a dependency for the privacy center (only `fides-js` is). I think the PC will probably need to import the IAB's TCF library at some point (when we want to have TCF notices in the privacy center) so I think it might make sense to get that type from there when the time comes. Although I'm not sure if we'll run into those webpack issues you were seeing when you tried to use their lib...

Reply:
ohhh I see, no worries here then.